packages/core/src/lib/stream_manager/utils.ts

```diff
@@ -0,0 +1,22 @@
+import type { Connection } from "@libp2p/interface";
+
+export function selectConnection(
```

Comment:
this was moved from `@waku/utils`

@waku-org/js-waku-developers let's be cautious of not placing non public utils in `@waku/utils`

non public utils are those that are needed only by `@waku/*` packages family and/or test package.